docker/indeaa-local.env

```diff
@@ -50,6 +50,12 @@ SUPERUSER_PASSWORD=Password123
 PGADMIN_DEFAULT_EMAIL=admin@admin.com
 PGADMIN_DEFAULT_PASSWORD=Password123
 
+# ======
+# GOOGLE
+# ======
+GOOGLE_CLIENT_ID=287885917936-040ee4p7ppe8krf1a0dh3nvaedm76qt1.apps.googleusercontent.com
+GOOGLE_SECRET=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
+
```

Comment:
yeah i suppose i don't mind having these environment variables right over here... 
BUT KEEP IN MIND to make this different from production

Reply:
Yep.
Should I put some placeholder text there instead? I have written in the docs that this needs to be changed before starting it

Reply:
i think thats all good